src/octoprint/server/api/folders.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function
+
+__author__ = "Gina Häußge <osd@foosel.net>"
+__license__ = 'GNU Affero General Public License http://www.gnu.org/licenses/agpl.html'
+__copyright__ = "Copyright (C) 2014 The OctoPrint Project - Released under terms of the AGPLv3 License"
+
+
+from flask import jsonify, make_response, request, url_for
+
+from octoprint.server.api import api
+from octoprint.server import printer, fileManager, slicingManager, eventManager, NO_CONTENT, current_user
+from octoprint.server.util.flask import restricted_access, get_json_command_from_request, with_revalidation_checking
+from octoprint.settings import settings, valid_boolean_trues
+
+import psutil
+import logging
+
+def _create_lastmodified(path, recursive):
+	if path.endswith("/files"):
+		# all storages involved
+		lms = [0]
+		for storage in fileManager.registered_storages:
```

Comment:
Huh? Why is the stuff in file manager part of the lastmodified value for the internal folders?

Reply:
Sorry, I am still getting familiar with Flask and the structure of OctoPrint so I copied a bunch of stuff from other places.